hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/DatanodeLayoutStorage.java

```diff
@@ -64,6 +64,11 @@ protected Properties getNodeProperties() {
     return new Properties();
   }
 
+  @Override
+  public void setClusterId(String clusterId) throws IOException {
+    super.getStorageInfo().setClusterId(clusterId);
```

Comment:
Thanks @nishitpatira for this patch.  We don't need to override the `setClusterId` method here.  There is already an implementation provided by the `abstract Storage class`.  Not needed here, just use the base class implementation.